api/strategies/openidStrategy.js

```diff
@@ -0,0 +1,78 @@
+const passport = require('passport');
+const express = require('express');
+const jwt = require('jsonwebtoken');
+const { Issuer, Strategy: OpenIDStrategy } = require('openid-client');
+const config = require('../../config/loader');
+const domains = config.domains;
+
+const User = require('../models/User');
+
+Issuer.discover(process.env.OPENID_ISSUER)
+  .then(issuer => {
+    const client = new issuer.Client({
+      client_id: process.env.OPENID_CLIENT_ID,
+      client_secret: process.env.OPENID_CLIENT_SECRET,
+      redirect_uris: [domains.server + process.env.OPENID_CALLBACK_URL]
+    });
+
+    const openidLogin = new OpenIDStrategy(
+      {
+        client,
+        params: {
+          scope: process.env.OPENID_SCOPE
+        }
+      },
+      async (tokenset, userinfo, done) => {
+        try {
+          let user = await User.findOne({ email: userinfo.email });
+          if (!user) {
+            user = new User({
+              provider: 'openid',
+              openidId: userinfo.sub,
+              username: userinfo.given_name,
+              email: userinfo.email,
+              emailVerified: userinfo.email_verified,
+              name: userinfo.given_name + ' ' + userinfo.family_name,
+              avatar: ''
+              // avatar: userinfo.picture
```

Comment:
This looks like valid/commented out code. Just checking if we can indeed pull in the user avatar? 
(x2 for the other time this occurs slightly further down)